assets/blocks/PineappleBush.block

```diff
@@ -0,0 +1,6 @@
+{
+  "hardness": 7,
+  "mass": 250,
```

Comment:
250 mass is *really* heavy. For comparison the standard plant template (`plant.block` under `Core/assets/blocks/templates`) has a mass of 5 (and a hardness of 1). Rocks and bricks have mass around 100.